### PR TITLE
[react-redux] Fix types for react-redux for connect without dispatchToProps

### PR DESCRIPTION
**Pull request by artem-malko**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <[#17829](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/17829)> <[#17829](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/17829)>
Actually, it is just copy of other pull-request [17876](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17876), but author closed that request.
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by artem-malko** [comment]
@thasner hello) I've opened new pull request with fix from [17876](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17876). Can we merge it?)

File: types/react-redux/index.d.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-// Type definitions for react-redux 4.4.0
+// Type definitions for react-redux 4.4.47
 // Project: https://github.com/rackt/react-redux
 // Definitions by: Qubo <https://github.com/tkqubo>, Sean Kelley <https://github.com/seansfkelley>, Thomas Hasner <https://github.com/thasner>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.4.1
```
Comment (thasner) [inline types/react-redux/index.d.ts]: you might wanna change this in a separate PR so you don't have to update whatever else is breaking the build

**Comment by thasner** [comment]
👍

**Comment by artem-malko** [comment]
@thasner so, what is the next step?)

**Comment by thasner** [comment]
@artem-malko "pending express review by a maintainer". That's usually pretty quick

**Comment by andy-ms** [comment]
Thanks!

**Comment by artem-malko** [comment]
@andy-ms could you tell me, when new version will be published?)

**Comment by andy-ms** [comment]
It just was.

**Comment by artem-malko** [comment]
👍

**Pull request merged**